monitoring/src/config.ts

```diff
@@ -30,6 +30,8 @@ export const getConfig = async (): Promise<LoopConfig> => {
   const domain = (await stackReference.getOutputValue('rootDomainName')) as string
   const additionalDomain = process.env.ADDITIONAL_ROOT_DOMAIN_NAME
 
+  console.log(`Extra domain: ${additionalDomain}`)
```

Comment:
Gm! Is it wanted?

Reply:
...not quite. I've left this here just to demonstrate an issue with the current CI workflow, where even thou our domains are seemingly injected correctly, the resulting ingress does not have that domain on the match list, resulting in a 404 once you visit https://monitoring.shapeshift.com/. Because this flow is only triggered on demand, I was trying to replicate the issue and see if there's some condition to the failures, so far with no lock. So short answer is no, but I'd keep this in until this issue is found and fixed

Reply:
Reverting this to make this PR posh, will work on this separately